<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefRecordDlg</class>
 <widget class="QWidget" name="DlgPrefRecordDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form3</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxRecordingsPath">
     <property name="title"><string>Recordings Directory</string></property>
     <layout class="QGridLayout" name="gridLayout_Recordings">
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel_Recordings">
        <property name="enabled"><bool>true</bool></property>
        <property name="minimumSize">
         <size><width>100</width><height>10</height></size>
        </property>
        <property name="font"><font/></property>
        <property name="text"><string>Recordings Directory:</string></property>
        <property name="wordWrap"><bool>false</bool></property>
        <property name="buddy" >
         <cstring>LineEditRecordings</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LineEditRecordings">
        <property name="minimumSize"> 
         <size><width>100</width><height>10</height></size>
        </property>
        <property name="font"><font/></property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="PushButtonBrowseRecordings">
        <property name="font"><font/></property>
        <property name="text"><string>Browse...</string></property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxEncoding">
     <property name="title">
      <string>Encoding</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxQuality">
     <property name="title">
      <string>Quality</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QLabel" name="TextQuality">
        <property name="text">
         <string>Quality</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>SliderQuality</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2" colspan="3">
       <widget class="QSlider" name="SliderQuality">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>11</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>9</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="TextLabel8_2_2_2">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>High</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="TextLabel8_3_2">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Low</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Tags</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel3_2_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>LineEditTitle</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LineEditTitle"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel3_2_3_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Author</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>LineEditAuthor</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="LineEditAuthor"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="TextLabel3_2_3_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Album</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>LineEditAlbum</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="LineEditAlbum"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="CheckBoxRecordCueFile">
        <property name="text">
         <string>Create a CUE file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>File Splitting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Split recordings at</string>
        </property>
        <property name="buddy" >
         <cstring>comboBoxSplitting</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBoxSplitting"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
